account_statement_import_online_ponto/models/ponto_buffer.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2022 Therp BV <https://therp.nl>.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+"""Define model to hold data retrieved from Ponto."""
+import json
+import logging
+
+from odoo import _, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class PontoBuffer(models.Model):
```

Comment:
@pedrobaeza requesting second opinion - it feels like models should've been prefixed, yet I'm not entirely sure what are the guidelines on that